core/src/main/java/me/mykindos/betterpvp/core/combat/listeners/CombatListener.java

```diff
@@ -472,6 +474,26 @@ private void playDamageEffect(CustomDamageEvent event) {
                 damagee.getWorld().playSound(damagee.getLocation(), sound.name().asString(), sound.volume(), sound.pitch());
             }
         }
+
```

Comment:
I dont mind the normal crit particles, but IMO the enchanted_hit ones look pretty bad. If you want to showcase high damage numbers, multiply the amount of crit particles by the amount of damage that exceeds 7.0. This will mean that basic attacks do not show particles, but as soon as abilities/passives/runes/legends are used that deal more damage than 7(the max with a power sword) the particles will show

Reply:
removed particles altogether